runtime/JavaScript/src/antlr4/Recognizer.d.ts

```diff
@@ -6,4 +6,7 @@ export declare class Recognizer<TSymbol> {
 
     removeErrorListeners(): void;
     addErrorListener(listener: ErrorListener<TSymbol>): void;
+    getErrorListenerDispatch(): ErrorListener<TSymbol>;
+    getLiteralNames(): string[] | [];
```

Comment:
I don't think the type is correct. The array cannot be empty and it contains `string | null `elements

Reply:
Same here

Reply:
I started from [this](https://github.com/antlr/antlr4/blob/b4c12ece2c6e44f4faf1f30c258e52c8142c2dcc/runtime/JavaScript/src/antlr4/Recognizer.js#L32).
I will correct it to:
```
    getLiteralNames(): string[];
    getSymbolicNames(): string[];
```